tools/prune-old-builds.py

```diff
@@ -57,9 +57,9 @@ def has_valid_name(folder_name: str) -> bool:
                         help="Delete folders.")
     args, _ = parser.parse_known_args()
 
-    builds_path = Path(args.builds_path)
-    repository_url = args.repository_url
-    repo_name = args.repo_name
+    builds_path = Path(pipes.quote(args.builds_path))
```

Comment:
Must fix: `pipes.quote` can change the string. (For example, it adds single quotes around a string if the string contains any spaces.) We only want to use `pipes.quote` when writing a shell script, not when calling functions such as `os.path.exists`.